src/components/webview/ErrorWebviewProvider.ts

```diff
@@ -46,23 +44,43 @@ export class ErrorWebviewProvider implements WebviewViewProvider {
 		);
 
 		messageBus.subscribe(MessageKind.clearState, () => {
-			__workspaceState.setSelectedCaseHash(null);
```

Comment:
other components might use this value from the `workspaceState`. Lets keep it for now